ubuntu/debian/patches/0001_arm_not_buildsamples_test.patch

```diff
@@ -0,0 +1,16 @@
+From: Jose Luis Rivero" <jrivero@osrfoundation.org>
+Subject: arm emulation is too slow to finish it on time
+
+diff --git a/test/integration/ExamplesBuild_TEST.cc b/test/integration/ExamplesBuild_TEST.cc
+index 8ae5f5e..a3245b8 100644
+--- a/test/integration/ExamplesBuild_TEST.cc
++++ b/test/integration/ExamplesBuild_TEST.cc
+@@ -373,7 +373,7 @@ void removeAll(const std::string &_path)
+ TEST(ExamplesBuild, Build)
+ {
+   // \todo(nkoenig) Fix windows.
+-#ifndef _WIN32
++#if defined(__arm__)
```

Comment:
I'm not sure about this logic. It looks like it is disabling the examples test unless `__arm__` is defined, which seems like the opposite of what I would have expected